pkg/resources/factory.go

```diff
@@ -112,6 +114,16 @@ func (rf *resourceFactory) GetResourcePool(rc *types.ResourceConfig, deviceList
 		}
 	}
 
+	// filter by linkTypes list
+	if rc.Selectors.LinkTypes != nil && len(rc.Selectors.LinkTypes) > 0 {
+		if len(rc.Selectors.LinkTypes) > 1 {
+			glog.Warningf("Link type selector should have a single value.")
```

Comment:
If this should be a single value,  why using a `list` instead of `string`?

Reply:
providing multiple link types will technically work (although i dont think its useful - hence the warning)
The reason i chose to keep it as a list of strings is two fold:
1. Keep consistent user API for specifying a selector
2. Supporting a single string value as a selector (instead of a list of strings) will require a refactor
    as it does not fit the current selector model.

Reply:
ok